test/e2e/multi-select.test.js

```diff
@@ -9,8 +9,115 @@ describe('vl-multi-select', async () => {
+        await multiselect.searchByPartialText('straat');
+        await assert.eventually.equal(multiselect.getNumberOfSearchResults(), 5);
+    });
+
+    it('Als gebruiker kan ik de values van een select opvragen', async () => {
+        const multiselect = await vlMultiSelectPage.getStandardMultiselect();
+        await assert.eventually.include(multiselect.values(), 'Belgium', 'France', 'Germany');
+    });
+
+    it('Als gebruiker kan ik controleren of een bepaalde value in de select beschikbaar is', async () => {
+        const multiselect = await vlMultiSelectPage.getStandardMultiselect();
+        await assert.eventually.isTrue(multiselect.hasValue('France'));
+    });
+
+    it('Als gebruiker kan ik luisteren naar een change event en onChange loggen naar de console', async () => {
+        const multiselect = await vlMultiSelectPage.getChangeEventMultiselect();
+        await multiselect.selectByValue('Germany');
+        await assert.eventually.isTrue(multiselect.hasAttribute('changed'));
+    });
+
+    it('Als gebruiker kan ik de multiselect via enable/disable-methode in/uit-schakelen', async () => {
+        const multiselect = await vlMultiSelectPage.getEnableDisableMethodeMultiselect();
+        await assert.eventually.isFalse(multiselect.isDisabled());
+        await vlMultiSelectPage.disable();
+        await assert.eventually.isTrue(multiselect.isDisabled());
+        await vlMultiSelectPage.enable();
+        await assert.eventually.isFalse(multiselect.isDisabled());
+    });
+
+    it('Als gebruiker kan ik programmatorisch (een) waarde(s) toevoegen/verwijderen in de multiselect', async () => {
+        const multiselect = await vlMultiSelectPage.getSetGetMethodeMultiselect();
+        await assert.eventually.isEmpty(multiselect.getSelectedOptionsByValue());
+        await vlMultiSelectPage.kiesDuitsland();
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'Germany');
+        await vlMultiSelectPage.verwijderSelectie();
+        await assert.eventually.isEmpty(multiselect.getSelectedOptionsByValue());
+        await vlMultiSelectPage.kiesBelgieEnNederland();
+        await assert.eventually.include(multiselect.getSelectedOptionsByValue(), 'Belgium', 'Netherlands');
+    });
+
+    it('Als een multiselect boven een datepicker gerenderd wordt, kunnen zowel de multiselect als de datepicker correct gebruikt worden', async () => {
+        const multiselect = await vlMultiSelectPage.getDatepickerMultiselect();
+        const datepicker = await vlMultiSelectPage.getDatepicker();
+        
+        const date = new Date();
+        const today = String(date.getDate()).padStart(1);
+        const dd = String(date.getDate()).padStart(2, '0');
+        const mm = String(date.getMonth() + 1).padStart(2, '0');
+        const yyyy = date.getFullYear();
+
+        await datepicker.selectDay(today);
+        await assert.eventually.equal(datepicker.getInputValue(), dd + '.' + mm + '.' + yyyy);
```

Comment:
Meer stijl/smaak, maar ik vind dit leesbaarder: 
```suggestion
        await assert.eventually.equal(datepicker.getInputValue(), `${dd}.${mm}.${yyyy}`);
```

Reply:
Kan ik mee leven.